covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -5,90 +5,136 @@
 from datetime import timedelta
 from flask import Flask
 from pathlib import Path
+from typing import List
 
 from covid_graphs.country_graph import CountryGraph, GraphType
-from covid_graphs.predictions import prediction_db, PredictionEvent
+from covid_graphs.country_report import CountryReport
+from covid_graphs.predictions import prediction_db, PredictionEvent, BK_20200411, OTHER
 
 
-def create_dashboard(
-        data_dir: Path,
-        server: Flask,
-        prediction_event: PredictionEvent,
+def create_graphs(
+    reports: List[CountryReport], prediction_event: PredictionEvent,
 ):
-    # TODO(miskosz): Don't use print.
-    print(f"Creating dashboard for {prediction_event.name} graphs.")
-
-    predictions = prediction_db.predictions_for_event(prediction_event)
+    report_from_name = {report.short_name: report for report in reports}
```

Comment:
Minor side note: My personal preference is "by" in place of "from". Either is fine though.

Reply:
Fixed.

I used to have 'by' there actually. But then when I said it out loud in my head it didn't sound good, so I changed it.

I almost always use 'by' in code in these situations though. Just today it didn't sound right.